### ISSUE TITLE
[req]: Manual Search webui: Allow us to save result filters as presets.

### ISSUE THREAD

**Issue by batuhankrmn** [Jackett/Jackett#13618]
### Is there already a request for your feature?

- [X] I have checked older issues, open and closed

### Is your feature request related to a problem? Please describe.

I always look for movies and TV shows from Jackett webui. Most frustrating scenario is I have SDR monitor, when I search for "Avengers Endgame 1080p" there will be tons of releases which are HDR/DoVi. Obviously, I can't watch them, so I have to filter the results every time with -hdr -dovi -dv -dolby.vision -dolbyvision -hdr10 -dolby to eliminate all results that contains these tags. It's really annoying to do that every time.

### Describe the solution you'd like

![image](https://user-images.githubusercontent.com/57859656/193430319-aaca4a96-800a-4b92-b5c4-59d5fedc10ad.png)
If we can save our filter parameters as preset to use next time, it'd be really useful for web ui. There could be dropdown menu such as "result filters", so we can filter our results selecting presets from this dropdown menu. Thanks for your consideration.

**Comment by garfield69** [comment]
v0.20.2080

**Comment by garfield69** [comment]
should we make this feature available on the cached releases page too?

**Comment by ilike2burnthing** [comment]
sounds good

**Comment by batuhankrmn** [comment]
@ilike2burnthing @garfield69 hey guys, just quick feedback here, I can't filter out the results with WEB-DL parameter. I guess it triggers negatives logic due to "-" operator.

**Comment by ilike2burnthing** [comment]
No, this is also happening with `.` and `_`, but not with `[`, so I'm very confused.

**Comment by garfield69** [comment]
issues with the search filter performance deserves its own ticket (as its precedes this enhancement), it is outside the scope of this ticket that deals with adding the preset enhancement.

**Comment by garfield69** [comment]
I will work on adding the preset to the cache page.

**Comment by garfield69** [comment]
bah. too complicated for me. the cache page js support is nothing like that used for the search results page.
PR welcome.

**Comment by ngosang** [comment]
This should be implemented after fixing => https://github.com/Jackett/Jackett/issues/13640#issuecomment-1374462683
Then it will be copy & paste

**Comment by PleXi00** [comment]
@batuhankrmn you have all of this on Radarr/Sonarr

**Comment by batuhankrmn** [comment]
> @batuhankrmn you have all of this on Radarr/Sonarr

Yeah but not interested with automated arr tools. I like the manual search of Jackett web UI. It's simple but powerful IMO. Especially If you're filtering the results a lot.
